cli/flags/common.go

```diff
@@ -61,6 +62,8 @@ func (commonOpts *CommonOptions) InstallFlags(flags *pflag.FlagSet) {
 
 	flags.StringVarP(&commonOpts.Username, "username", "u", "", fmt.Sprintf(`API username (will override %s env variable value)`, cliconfig.EnvStorageosUsername))
 	flags.StringVarP(&commonOpts.Password, "password", "p", "", fmt.Sprintf(`API password (will override %s env variable value)`, cliconfig.EnvStorageosPassword))
+
+	flags.IntVarP(&commonOpts.Timeout, "timeout", "t", 0, fmt.Sprintf(`client timeout in seconds (will override %s env variable value if set), default 10s`, cliconfig.EnvStorageOSTimeout))
```

Comment:
I'm pretty sure the 4th field is the default value, and I would imagine the default value is included in the output automatically

```suggestion
flags.IntVarP(&commonOpts.Timeout, "timeout", "t", cliconfig.DefaultTimeout, fmt.Sprintf(`client timeout in seconds (will override %s env variable value if set)`, cliconfig.EnvStorageOSTimeout))
```

However we should swap this to:

```suggestion
	help := fmt.Sprintf("client timeout in seconds (will override %s env variable value if set)", cliconfig.EnvStorageOSTimeout)
	flags.DurationVarP(p *time.Duration, "timeout", "t", cliconfig.DefaultTimeout, help)
```

Reply:
The default value is from environmental variable, so it should not be set from command line argument. Same logic as controlplane.

Reply:
Can't we make use of `flags.Changed()` like in the func below? Seems like we're working against the CLI library here, that's all.

Reply:
done